src/components/GamePage.vue

```diff
@@ -125,69 +125,75 @@ keys["Alt"] = "Alt";
 keys["Meta"] = "Meta";
 keys[" "] = "space";
 
-
+//使いわますため（次のインデックスにするためのやつ）
+function substfunc(ind:num) {
+  code.correctcode = code.fullcode.substring(0,ind)
+  code.pointercode = code.fullcode.substring(ind,ind+1);
+  code.notyetcode = code.fullcode.substring(ind+1);
+}
 let index = 0;
```

Comment:
リアクティブな変数にするために`ref()`関数を使用してください
```suggestion
let index = ref<number>(0);
```